packages/spec-api/test/expectation.test.ts

```diff
@@ -0,0 +1,50 @@
+import { expect } from "chai";
+import "mocha";
```

Comment:
remove mocha and chai and import things from `vitest`

Reply:
Done